### PR TITLE
wrapped tests for describe_na_value in function

### PR DESCRIPTION
**Pull request by v5y8**

### CODE REVIEW

**Comment by v5y8** [comment]
closes #42

File: tests/testthat/test-eda.R [hunk 1]
```diff
@@ -1,69 +1,95 @@
...
-    col3 = c(NA_character_, "b")
-  )
 
-  not_a_tbl <- list(col1 = c(1, 2),
-                    col2 = c(0.5, 0.4),
-                    col3 = c("a", "b"))
 
+test_that("multiplication works", {
+  expect_equal(2 * 2, 4)
+})
```
Comment (marvinmin) [inline tests/testthat/test-eda.R]:
```suggestion
})
```

File: tests/testthat/test-eda.R [hunk 2]
```diff
@@ -1,69 +1,95 @@
...
 
-  expect_error(describe_na_values(not_a_tbl),
-               regexp = "The value of the argument 'dataframe' should be of type  'data.frame' or 'tibble'.")
+#' Tests the describe_na_value function
+#'
+#' @return None. the function will not throw an error
+#' if the tests fail.
+#' @export
```
Comment (marvinmin) [inline tests/testthat/test-eda.R]:
```suggestion
```
We don't need to export the test function to the NAMESPACE.

File: tests/testthat/test-eda.R [hunk 3]
```diff
@@ -1,69 +1,95 @@
...
+#'
+#' @examples
+#' test_na_value()
```
Comment (marvinmin) [inline tests/testthat/test-eda.R]:
```suggestion
#' test_describe_na_values ()
```

File: tests/testthat/test-eda.R [hunk 4]
```diff
@@ -1,69 +1,95 @@
...
+      )
+    ))
+
+
+  })
+
+}
+
+
+
```
Comment (marvinmin) [inline tests/testthat/test-eda.R]:
```suggestion
test_describe_na_values ()
```

**Review by marvinmin — CHANGES_REQUESTED** [review]
Only some minor changes are needed.

File: tests/testthat/test-eda.R [hunk 5]
```diff
@@ -1,69 +1,92 @@
...
-    col1 = c(1, 2),
-    col2 = c(0.5, 0.4),
-    col3 = c(NA_character_, "b")
-  )
 
-  not_a_tbl <- list(col1 = c(1, 2),
-                    col2 = c(0.5, 0.4),
-                    col3 = c("a", "b"))
 
+})
```
Comment (marvinmin) [inline tests/testthat/test-eda.R]:
```suggestion
```

File: tests/testthat/test-eda.R [hunk 6]
```diff
@@ -1,69 +1,92 @@
...
+test_describe_na_values ()
 
 #' Tests the correlation function to make sure output is rendered correctly
 #' or the function will fail with an error message and problem.
 #'
 #' @return None. the function will not throw an error
 #' if the tests fail.
 #'
+#' @export
+#'
```
Comment (marvinmin) [inline tests/testthat/test-eda.R:78]:
```suggestion
```

**Review by marvinmin — CHANGES_REQUESTED** [review]
only a little to change

**Comment by v5y8** [comment]
I've made the changes; please take a look.

**Review by marvinmin — APPROVED** [review]

**Pull request merged**